specification/hybridnetwork/resource-manager/Microsoft.HybridNetwork/stable/2024-04-15/publisher.json

```diff
@@ -0,0 +1,1714 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request is successful. The operation returns the resulting ArtifactStore resource.",
+            "schema": {
+              "$ref": "#/definitions/ArtifactStore"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a artifact store resource": {
+            "$ref": "./examples/ArtifactStoreGet.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "ArtifactStores"
+        ],
+        "operationId": "ArtifactStores_Update",
+        "description": "Update artifact store resource.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./common.json#/parameters/PublisherNameParameter"
+          },
+          {
+            "$ref": "./common.json#/parameters/ArtifactStoreParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./common.json#/definitions/TagsObject"
+            },
+            "description": "Parameters supplied to the create or update application group operation."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Creation or update is successful. The operation returns the resulting ArtifactStore sub resource.",
+            "schema": {
+              "$ref": "#/definitions/ArtifactStore"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update artifact store resource tags": {
+            "$ref": "./examples/ArtifactStoreUpdateTags.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HybridNetwork/publishers/{publisherName}/artifactStores/{artifactStoreName}/addNetworkFabricControllerEndPoints": {
```

Comment:
>addNetworkFabricControllerEndPoints [](http://example.com/codeflow?start=168&length=35)

Add examples for all the new APIs?

Reply:
We have already added examples for all new APIs introduced in this version, for example,
specification/hybridnetwork/resource-manager/Microsoft.HybridNetwork/stable/2024-04-15/examples/ArtifactStoreAddNFCEndPoints.json
specification/hybridnetwork/resource-manager/Microsoft.HybridNetwork/stable/2024-04-15/examples/ArtifactStoreApprovePrivateEndPoints.json

and it is referred in x-ms-examples in Publisher.json

Reply:
I was comparing commit 5 with commit 6 and these are in commit 5. That's why these example files don't show up.